core/array/any_spec.rb

```diff
@@ -0,0 +1,30 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Array#any?" do
+  describe 'with no block given' do 
+    it "checks if there are any members of an array" do 
```

Comment:
I didn't catch it for Hash#any?, but it is actually the same (could you fix in a separate commit?):
`any?` without block behaves as if you passed `{ |e| e }` so for instance `[nil].any?` is `false`.